octodns/record/__init__.py

```diff
@@ -217,6 +217,16 @@ def healthcheck_port(self):
         except KeyError:
             return 443
 
+    @property
+    def healthcheck_response(self):
+        healthcheck = self._octodns.get('healthcheck', {})
+        if healthcheck.get('protocol', None) == 'TCP':
+            return None
+        try:
+            return healthcheck['response']
+        except KeyError:
+            return {'contains': '200 OK'}
```

Comment:
This is specific to NS1 so doesn't belong in the non-provider specific/generic Record object